Documentation/CRDs/Cluster/network-providers.md

```diff
@@ -0,0 +1,191 @@
+---
+title: CephCluster Network Providers
```

Comment:
We're already in the cluster section for the TOC, seems like we can simplify the title
```suggestion
title: Network Providers
```